docs_src/tutorial_1_hello_world.py

```diff
@@ -0,0 +1,20 @@
+from corva import Api, Corva, Event, State
```

Comment:
from corva import Corva is a little odd. Can we rename that container?

Reply:
maybe we can leave it as it is. other popular libraries use the same approach e.g.
```python3
from celery import Celery
from fastapi import FastAPI
```
if not, what would be a better naming?